menu/fail2ban_menu.sh

```diff
@@ -44,8 +45,32 @@ case "$choice" in
             msg_box "Fail2ban isn't installed. Please run 'sudo bash /var/scripts/menu.sh' to install it." "$SUBTITLE"
         fi
     ;;&
+    *"Fail2ban-UnbanIP"*)
+        SUBTITLE="Fail2ban Unban IP"
+        if is_this_installed fail2ban && [ -f "/etc/fail2ban/filter.d/nextcloud.conf" ]
+        then
+            UNBANIP="$(input_box_flow "Enter the IP address that you want to unban.")" "$SUBTITLE"
+            if ! iptables -L -n | grep -qv "$UNBANIP"
```

Comment:
This needs tweaking.

We should at least check if the IP is banned before trying to unban it. @szaimen